Makefile

```diff
@@ -119,9 +129,12 @@ mixedindentlint: node_modules/mixedindentlint
 	@$(RECORD_ENV) $@
 
 # generate the client-side `config` js file
-$(CLIENT_CONFIG_FILE): .env config/$(CALYPSO_ENV).json config/client.json client/config/regenerate.js
+$(CLIENT_CONFIG_FILE): .env config/$(CALYPSO_ENV).json config/client.json client/config/regenerate.js client/lib/user/shared-utils.js
 	@$(NODE) client/config/regenerate.js > $@
 
+client/lib/user/shared-utils.js: server/user-bootstrap/shared-utils.js
+	@ln -s $(realpath $<) $@
+
```

Comment:
Unfortunately, no version of Git for Windows that I've found [supports symlinks](http://stackoverflow.com/a/16754068/181497). So I've removed the symlink in git for `client/lib/user/shared-utils.js` and instead create it via `make`.